src/task_types/multiversion.rs

```diff
@@ -94,4 +109,113 @@ impl MultiversionService for MultiversionServiceImpl {
 
         tags.join(",")
     }
+
+    /// Get the filtered multiversion generate tasks based on the config.
+    ///
+    /// # Arguments
+    ///
+    /// * `multiversion_generate_tasks` - Array of desired tasks to generate.
+    ///
+    /// # Returns
+    ///
+    /// Multiversion generate tasks that match the config's `last versions`.
+    fn filter_multiversion_generate_tasks(
```

Comment:
`multiversion_config` is the output from running `buildscripts/resmoke.py multiversion-config`.
ie:
```
last_continuous_fcv: '7.0'
last_lts_fcv: '7.0'
last_versions:
- last_lts
requires_fcv_tag: requires_fcv_71
requires_fcv_tag_continuous: requires_fcv_71
requires_fcv_tag_lts: requires_fcv_71
```
When last-lts == last-continuous only last-lt is present in `last_versions`. So we filter the `multiversion_generate_tasks` and make sure that each multiversion task's "old version" exists in `last_versions`